hazelcast/src/main/java/com/hazelcast/map/impl/MapContainerSupport.java

```diff
@@ -26,7 +26,7 @@
  *
  * @see MapContainer
  */
-abstract class MapContainerSupport {
+public abstract class MapContainerSupport {
```

Comment:
I had to make this class `public` since I was forbidden to mock a method from it otherwise.